docs/news/news-v5.0.x.rst

```diff
@@ -0,0 +1,282 @@
+  Users can also use environment variable ``AUTOMAKE_JOBS``.
+- PMI support has been removed for Open MPI apps.
+- Legacy btl/sm has been removed, and replaced with btl/vader, which
+  was renamed to btl/sm.
+- Update btl/sm to not use CMA in user namespaces.
+- C++ bindings have been removed.
+- The ``--am`` and ``--amca`` options have been deprecated.
+- opal/mca/threads framework added. Currently supports
+  argobots, qthreads, and pthreads. See the --with-threads=x option
+  in configure.
+- Various ``README.md`` fixes - thanks to:
+  Yixin Zhang <zhany217@wfu.edu>,
+  Samuel Cho <choss@wfu.edu>,
+  Robert Langfield <langrc18@wfu.edu>,
+  Alex Ross <rossaj16@wfu.edu>,
+  Sophia Fang <fangq18@wfu.edu>,
+  mitchelltopaloglu <mitchelltopaloglu@gmail.com>,
+  Evstrife <wus217@wfu.edu>, and
+  Hao Tong <tongh18@gemini.deac.wfu.edu> for their
+  contributions.
+- osc/pt2pt: Removed. Users can use osc/rdma + btl/tcp
+  for OSC support using TCP, or other providers.
+- Open MPI now links -levent_core instead of -levent.
+- common/cuda docs: Various fixes. Thanks to
+  Simon Byrne <simonbyrne@gmail.com> for finding and fixing.
+- osc/ucx: Add support for acc_single_intrinsic.
+- Fixed ``buildrpm.sh -r`` option used for RPM options specification.
+  Thanks to John K. McIver III <john.mciver.iii@gmail.com> for
+  reporting and fixing.
+- configure: Added support for setting the wrapper C compiler.
+  Adds new option ``--with-wrapper-cc=``.
+- mpi_f08: Fixed Fortran-8-byte-INTEGER vs. C-4-byte-int issue.
+  Thanks to @ahaichen for reporting the bug.
+- opal/thread/tsd: Added thread-specific-data (tsd) api.
+- Add missing ``MPI_Status`` conversion subroutines:
+  ``MPI_Status_c2f08()``, ``MPI_Status_f082c()``, ``MPI_Status_f082f()``,
+  ``MPI_Status_f2f08()`` and the ``PMPI_*`` related subroutines.
+- patcher: Removed the Linux component.
+- opal/util: Fixed typo in error string. Thanks to
+  NARIBAYASHI Akira <a.naribayashi@fujitsu.com> for finding
+  and fixing the bug.
+- Fortran/use-mpi-f08: Generate PMPI bindings from the MPI bindings.
+- Converted man pages to markdown.
+  Thanks to Fangcong Yin <fyin2@nd.edu> for their contribution
+  to this effort.
+- Fixed ompi_proc_world error string and some comments in pml/ob1.
+  Thanks to Julien EMMANUEL <julien.emmanuel@inria.fr> for
+  finding and fixing these issues.
+- oshmem/tools/oshmem_info: Fixed Fortran keyword issue when
+  compiling param.c. Thanks to Pak Lui <pak.lui@amd.com> for
+  finding and fixing the bug.
+- autogen.pl: Patched libtool.m4 for OSX Big Sur. Thanks to
+  @fxcoudert for reporting the issue.
+- Updgraded to HWLOC v2.4.0.
+- Removed config/opal_check_pmi.m4.
+  Thanks to Zach Osman <zosman@gmu.edu> for the contribution.
+- opal/atomics: Added load-linked, store-conditional atomics for
+  AArch6.
+- Fixed envvar names to OMPI_MCA_orte_precondition_transports.
+  Thanks to Marisa Roman <marisa.roman@cornelisnetworks.com>
+  for the contribution.
```

Comment:
```suggestion
- Fixed environment variables names to ``OMPI_MCA_orte_precondition_transports``.
```

What does this sentence mean?

Marissa works for a core community member; we don't need to cite her.

Reply:
They must have been added since that PR, at that time they were not. Will remove.